app/Broadcasts/LogSmash/Export/LogExportBroadcast.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace Nexus\ApexEvents\Broadcasts\LogSmash\Export;
+
+use Nexus\ApexEvents\Broadcasts\AbstractBroadcast;
+
+/**
+ * Class LogExportBroadcast
+ * @package Nexus\ApexEvents\Broadcasts\LogSmash
+ */
+class LogExportBroadcast extends AbstractBroadcast
+{
+    /**
+     * @var string
+     */
+    public $guid;
+
+    /**
+     * @var string
+     */
+    public $environment;
+
+    /**
+     * @var string
+     */
+    public $timestamp;
+
+    /**
+     * @var array
+     */
+    public $tenant;
+
+    /**
+     * @var string
+     */
+    public $type;
+
+    /**
+     * @var string
+     */
+    public $method;
+
+    /**
+     * @var array
+     */
+    public $request;
+
+    /**
+     * LogExportBroadcast constructor.
+     * @param string $guid
+     * @param string $environment
+     * @param string $timestamp
+     * @param array  $tenant
+     * @param string $type
+     * @param string $method
+     * @param array  $request
+     */
+    public function __construct(
+        string $guid,
+        string $environment,
+        string $timestamp,
+        array $tenant,
+        string $type,
+        string $method,
+        array $request
+    ) {
+        $this->guid        = $guid;
+        $this->environment = $environment;
+        $this->timestamp   = $timestamp;
+        $this->tenant      = $tenant;
+        $this->type        = $type;
+        $this->method      = $method;
+        $this->request     = $request;
```

Comment:
```suggestion
```

These fields aren't relevant to exports, these were specifically for API requests.

There need to be fields for the export, such as export name, stage (of the export process), etc...